src/simple/multiplex/mod.rs

```diff
@@ -91,12 +91,13 @@ mod lift {
         }
     }
 
-    impl<T, InnerItem, InnerSink, E> Transport<()> for LiftTransport<T, E> where
-        E: 'static,
-        T: 'static,
-        T: Stream<Item = (RequestId, InnerItem), Error = io::Error>,
-        T: Sink<SinkItem = (RequestId, InnerSink), SinkError = io::Error>
-    {}
+    impl<T, InnerItem, InnerSink, E> Transport<()> for LiftTransport<T, E>
+        where E: 'static,
+              T: 'static,
+              T: Stream<Item = (RequestId, InnerItem), Error = io::Error>,
+              T: Sink<SinkItem = (RequestId, InnerSink), SinkError = io::Error>
+    {
+}
```

Comment:
Nice catch! I should have checked the output more carefully. This definitely looks like a bug in rustfmt. I'm trying to find an minimal repro so I can open an issue with them.